src/popup/components/TransactionTagList.vue

```diff
@@ -127,6 +128,13 @@ export default defineComponent({
       }
       if (txType.value === TX_TYPE_MDW.PayingForTx) {
         return [
+          i18n.t('transaction.type.createMultisigVault'),
```

Comment:
This is the case only if inner transaction is `GAAttachTx` and we are actually using this multisig contract